src/binary-reader-objdump.cc

```diff
@@ -23,6 +23,10 @@
 #include <cstring>
 #include <vector>
 
+#if HAVE_STRCASECMP
+#include <strings.h>
+#endif
```

Comment:
Given that we don't guard use of strcasecmp why guard the include?

Reply:
On MSVC, it's polyfilled with `_strcmpi` which is in `<string.h>` and it's included in `"config.h"` thus no additional include required.

https://github.com/WebAssembly/wabt/blob/4ba97c116d1417b2fb516299a37965820b93ced3/src/config.h.in#L288

I choose not to include `<strings.h>` in `"config.h"` because it's not a standard C header.